lib/order.rb

```diff
@@ -0,0 +1,33 @@
+require_relative 'dish'
+
+class Order
+
+	attr_reader :current_order
+
+	def initialize
+		@current_order = []
+	end
+
+	def update_order(item)
+		@current_order << item
+	end
+
+	def confirm_order
+		total_price = @current_order.inject(0) {|sum,item| sum += item.price }.round(2)
+		puts "Is the following order correct? y/n"
+		@current_order.each { |item| puts "#{item.name} #{item.price}" }
+		puts "£#{total_price}"
+		input = STDIN.gets.chomp
+		input.downcase == "y" ? send_text : input.downcase == "n" ? puts("Sorry. Please review your order.") : puts("please enter y or n")
+	end
+
+	def send_text
```

Comment:
The order shouldn't be responsible for sending text messages.  Do you have a missing object?